### PR TITLE
Add ansible cloud team to tower managers

### PR DESCRIPTION
**Pull request by jillr**
Add Ansible cloud team members and validated content repo names

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -457,6 +467,9 @@ orgs:
           aap_configuration_template: admin
           aap_utilities: admin
           ah_configuration: admin
+          cloud.aws_ops: admin
```
Comment (sabre1041) [inline config.yaml:470]: These repositories are not yet available

**Review by sabre1041 — CHANGES_REQUESTED** [review]

File: config.yaml (hunk 1 above)
Comment (jillr) [reply config.yaml:470]: Correct, they need to be migrated from ansible-collections, my understanding was the process was easier if I had this PR pre-prepared

**Comment by jillr** [comment]
Removed the azure roles repo; we'll add that in a follow-up PR as it will take longer to update it for new naming guidance.
ready for review

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**